imagetagger/imagetagger/images/migrations/0016_settag_test.py

```diff
@@ -0,0 +1,30 @@
+# Generated by Django 2.0.7 on 2018-07-10 22:30
+
+from django.db import migrations
+
+
+def set_test_tag(apps, schema_editor):
+    SetTag = apps.get_model('images', 'SetTag')
+    if SetTag.objects.filter(name='test').exists():
```

Comment:
For better adaptability (e.g., to support multiple database), you should use an explicit using=schema_editor.connection.alias wherever you use the database. That is for all model.objects and all model object operations (save, delete, etc.).